### DISCUSSION TITLE
13 - Curated Schemas

### DISCUSSION THREAD

**Discussion by kevinslin** [dendronhq/dendron#735] [RFCs/Ideas]
Dendron should have an easily accessible repo for people to download and share schemas

**Comment by ScriptAutomate** [comment]
I wanted to follow-up with this discussion, using the following links:
- Official Dendron repo: [Templates](https://github.com/dendronhq/templates)
- Official Dendron repo: [Schema Library](https://github.com/dendronhq/schema-library)
- [`awesome-dendron`](https://github.com/dendronhq/awesome-dendron/): Awesome list includes a section of [public vaults](https://github.com/dendronhq/awesome-dendron/#community-public-vaults), such as [Bassman's Dendron Schemas](https://github.com/Bassmann/Dendron-schemas)
